tests/mlmodel_sklearn/test_feature_events.py

```diff
@@ -18,6 +18,7 @@
 import pandas
```

Comment:
Perhaps we should rename this test file to `test_inference_events.py` since it tests both label and feature events now.